packages/web3/src/transaction/sign-verify.test.ts

```diff
@@ -46,4 +46,117 @@ describe('transaction', function () {
     const signature = transactionSign(txHash, privateKey)
     expect(transactionVerifySignature(txHash, publicKey, signature)).toEqual(true)
   })
+
+  describe('transactionVerifySignature additional tests', () => {
+    const validTxHash = '8fc5f0d120b730f97f6cea5f02ae4a6ee7bf451d9261c623ea69d85e870201d2'
```

Comment:
Maybe we can 
1. Use `randomTxId()` function to create a `validTxId`
2. Create `validPubKey` using `PrivateKeyWallet.Random`
3. Generate the `validSignature`

So we don't need to hardcode value here

Reply:
Ok sure. Thanks

Reply:
since I can't find a randomTxId() function, I actually don't know if there is a randomTxId() function somewhere.  Hope I can create my own generateRandomTxId() function to achieve a good implementation

Reply:
Its in `import { randomTxId } from '../contract'`.